File: src/main/java/me/dreamhopping/pml/api/events/impl/client/chat/ClientChatSentEvent.java
package me.dreamhopping.pml.api.events.impl.client.chat;

import me.dreamhopping.pml.api.events.Event;

/**
 * Called when a chat message is sent by the client
 *
 * @author dreamhopping
 * @see Event
 */
public class ClientChatSentEvent extends Event {
    public String message;

    public ClientChatSentEvent(String message) {
        this.message = message;
    }
}

File: src/main/java/me/dreamhopping/pml/api/events/impl/client/chat/ClientChatSentEvent.kt
package me.dreamhopping.pml.api.events.impl.client.chat

import me.dreamhopping.pml.api.events.Event

/**
 * Called when a chat message is sent by the client
 *
 * @author dreamhopping
 * @see Event
 */
class ClientChatSentEvent(var message: String) : Event()